libs/core/input-group/input-group.component.html

```diff
@@ -28,7 +28,7 @@
                     [fdType]="inShellbar ? 'standard' : 'transparent'"
                     [id]="_addOnButtonId"
                     [attr.tabindex]="buttonFocusable ? 0 : -1"
-                    [attr.title]="iconTitle || glyphAriaLabel || glyph"
+                    [attr.title]="iconTitle || glyphAriaLabel"
```

Comment:
I am sorry, I don't understand how this is fixing the issue in Value Help Dialog. If the user doesn't pass title or aria-label for the icon, the Button will have no title attribute.

Reply:
The issue isn't with value help dialog, but multi input. Also, the accessibility designers have advised we move away from using title attribute because it's not keyboard/screenreader friendly. You won't see a title on the [ui5 multi input button](https://sapui5.hana.ondemand.com/#/entity/sap.ui.comp.smartmultiinput.SmartMultiInput/sample/sap.ui.comp.sample.smartmultiinput.withBinding). I'm leaving the inputs for now for the purpose of avoiding breaking changes